nomad/vault.go

```diff
@@ -1305,7 +1311,7 @@ func (v *vaultClient) revokeDaemon() {
 					revoking = append(revoking, va)
 				}
 
-				if len(revoking) >= toRevoke {
+				if len(revoking)+len(ttlExpired) >= toRevoke {
```

Comment:
Maybe a comment indicating that `maxVaultRevokeBatchSize` is meant to constraint the batch size for both submitting requests to Vault as well as restrict the size of Raft messages - hence needing to account for ttlExpired too.